lib/keyboard_utils.coffee

```diff
@@ -49,6 +49,8 @@ KeyboardUtils =
       event.key.toLowerCase() # F1 to F9.
     else if event.key.length == 3 and "F10" <= event.key <= "F12"
       event.key.toLowerCase() # F10 to F12.
+    else if event.key == "Backspace"
+      event.key.toLowerCase() # Backspace.
```

Comment:
These two lines has been removed, so may the `Changes requested` flag be removed?